packages/client/src/Accounts.tsx

```diff
@@ -1,15 +1,87 @@
 import { Link, RouteComponentProps } from "@reach/router"
 import * as React from "react"
 import * as graphql from "./generated/graphql"
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles"
+import DeleteIcon from "@material-ui/icons/Delete"
+import {
+  AppBar,
+  Fab,
+  Typography,
+  TextField,
+  Card,
+  CardHeader,
+  IconButton,
+  CardActions,
+  Button,
+  Toolbar,
+  Modal
+} from "@material-ui/core"
+import NavigationIcon from "@material-ui/icons/Navigation"
+import AddIcon from "@material-ui/icons/Add"
+import CloseIcon from "@material-ui/icons/Close"
+import { ThemeProvider } from "@material-ui/styles"
 
+const useStyles = makeStyles(theme => ({
+  fab: {
+    margin: 0,
+    position: "fixed",
+    right: theme.spacing(2),
+    bottom: theme.spacing(2),
+    zIndex: theme.zIndex.snackbar
+  },
+  appBarSpacer: theme.mixins.toolbar as any,
+  root: {
+    display: "flex",
+    flexDirection: "row",
+    flexWrap: "wrap",
+    justifyContent: "spaceAround",
+    alignContent: "spaceAround"
```

Comment:
When writing this CSS-in-JS the rule names are camel-cased, but the values have dashes as usual. The values here should be `"space-around"`, not `"spaceAround"`. If you were having trouble getting the layout to work properly that might be why.